extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -20,7 +20,7 @@ package stackdriver.config.v1alpha1;
 import "google/protobuf/duration.proto";
 
 message PluginConfig {
-  // next id: 8
+  // next id: 5
```

Comment:
Do we need to reserve these deprecated fields?

Reply:
It's fine, this is no user that actually starts to use this proto yet.